### DISCUSSION TITLE
2009/08/05/%E3%80%8A%E6%88%91%E5%92%8Clabview%E3%80%8B%E5%85%A8%E4%B9%A6%E7%9B%AE%E5%BD%95

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#272] [Announcements]
# 2009/08/05/%E3%80%8A%E6%88%91%E5%92%8Clabview%E3%80%8B%E5%85%A8%E4%B9%A6%E7%9B%AE%E5%BD%95

营养药和美容护肤品

https://blog.qizhen.xyz/2009/08/05/%E3%80%8A%E6%88%91%E5%92%8Clabview%E3%80%8B%E5%85%A8%E4%B9%A6%E7%9B%AE%E5%BD%95

**Comment by ruanqizhen** [comment]
之前博客下的留言：

飚
2009年09月3日 18:30 编辑
强,快快上市吧,等候通知中!!!

回复
志勇
2009年08月28日 12:46 编辑
强人，有书了，也通知我一下

回复
Qizhen
2009年08月28日 09:10 编辑
这本书还未上市。等有的卖了再通知大家。

回复
虹
2009年08月28日 06:42 编辑
您好阮工我对您的《我和LabVIEW 》一书非常赶兴趣，请告知如何购买。谢谢！张虹Hong.zhang@agresearch.co.nz

回复
家伟
2009年08月6日 13:55 编辑
楼主厉害,labview玩的都可以出书了, 强个沙发先,赞一个!

回复
